src/amo/reducers/collections.js

```diff
@@ -697,6 +703,32 @@ export const changeAddonCollectionsLoadingFlag = ({
   };
 };
 
+type LocalizeCollectionDetailParams = {|
+  detail: ExternalCollectionDetailWithLocalizedStrings,
+  lang: string,
+|};
+
+export const localizeCollectionDetail = (
+  { detail, lang }: LocalizeCollectionDetailParams
+): ExternalCollectionDetail => {
+  invariant(detail, 'detail is required for localizeCollectionDetail');
+  invariant(lang, 'lang is required for localizeCollectionDetail');
+
+  return {
+    addon_count: detail.addon_count,
+    author: detail.author,
+    default_locale: detail.default_locale,
+    description: detail.description && detail.description[lang],
```

Comment:
What is `description` if there is no `detail.description`?

Reply:
If `detail.description` is `null` then `description` will be `null`. If `detail.description` is `undefined` then `description` will be `undefined`.  What other case are we concerned about?

Reply:
Ok, then I would rather see a ternary operator here, because we only want `null` or a map.

Reply:
Ok, fixed.